d2/src/main/java/com/linkedin/d2/jmx/RelativeLoadBalancerStrategyJmx.java

```diff
@@ -0,0 +1,189 @@
+/*
+   Copyright (c) 2020 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package com.linkedin.d2.jmx;
+
+import com.linkedin.d2.balancer.clients.TrackerClient;
+import com.linkedin.d2.balancer.strategies.LoadBalancerQuarantine;
+import com.linkedin.d2.balancer.strategies.relative.RelativeLoadBalancerStrategy;
+import com.linkedin.d2.balancer.strategies.relative.StateUpdater;
+import com.linkedin.d2.balancer.strategies.relative.TrackerClientState;
+import com.linkedin.d2.balancer.util.partitions.DefaultPartitionAccessor;
+import com.linkedin.util.degrader.CallTracker;
+import java.net.URI;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+
+public class RelativeLoadBalancerStrategyJmx implements RelativeLoadBalancerStrategyJmxMBean
+{
+  private final RelativeLoadBalancerStrategy _strategy;
+
+  public RelativeLoadBalancerStrategyJmx(RelativeLoadBalancerStrategy strategy)
+  {
+    _strategy = strategy;
+  }
+
+  @Override
+  public double getLatencyStandardDeviation()
+  {
+    Map<TrackerClient, TrackerClientState> stateMap =
+        _strategy.getPartitionState(DefaultPartitionAccessor.DEFAULT_PARTITION_ID).getTrackerClientStateMap();
+
+    return calculateStandardDeviation(stateMap.keySet());
+  }
+
+  @Override
+  public double getLatencyMeanAbsoluteDeviation()
```

Comment:
As we discussed offline, I think it's fair to remove mean, since the calculation is a bit more complicated, also it did not provide too much additional value